<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManWindow</class>
 <widget class="QDialog" name="ManWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>545</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,1,1,0,10,1">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>添加汽车品牌</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,1,3">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>　品牌名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="maEd1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>　品牌拼音</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="maEd2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,1,3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>　区域语言</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="maEd3"/>
     </item>
     <item>
      <widget class="QRadioButton" name="maEd4">
       <property name="text">
        <string>IsNative</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3,1,3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>　　　Mem</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="maEd5"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="maEd6_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="maEd6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="maBn1">
       <property name="text">
        <string>保　存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="maBn2">
       <property name="text">
        <string>取　消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
